src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const getCurrentTab = (tab) => {
```

Comment:
give better naming to the function
```suggestion
  const onTabSelect = (tab) => {
```